lib/config.js

```diff
@@ -113,8 +113,9 @@ config.filesRoot = '/files';
 config.workspaceS3Bucket = 'workspaces';
 config.workspaceDevHostInstanceId = 'devHost1';
 config.workspaceDevHostHostname = 'localhost';
-config.workspaceDevHostPort = 8081;
+config.workspaceHostPort = 8081;
 config.workspaceDevContainerHostname = 'host.docker.internal';
+config.workspaceJobsDirectory = '/jobs';
```

Comment:
```suggestion
config.workspaceJobsDirectory = process.cwd();
```
Should this be `cwd`? That was the original default.